### PR TITLE
Update rippled-example.cfg for new node_db settings

### PR DESCRIPTION
**Pull request by vinniefalco**

### CODE REVIEW

**Comment by mtrippled** [comment]
:+1:

**Comment by nbougalis** [comment]
:+1:

File: doc/rippled-example.cfg [hunk 1]
```diff
@@ -606,30 +606,49 @@
...
+#       NuDB is high performance, but only works with Solid State (SSD)
+#       Drives or installations that have higher IOPS. It maintains its
+#       high speed regardless of the amount of history stored. Online
+#       delete may be selected, but is not required. NuDB is available
+#       on all platforms that rippled runs on.
+#
+#   type = RocksDB
+#
+#       This can work with spinning disks or installations that have
+#       lower IOPS (read/write throughput). It is not suitable for
```
Comment (rec) [inline doc/rippled-example.cfg]: If you need to define IOPS, you should do it the first time you mention it (above).

File: doc/rippled-example.cfg [hunk 2]
```diff
@@ -606,30 +606,49 @@
...
-#       SQLite              (Deprecated)
-#       none                (No backend)
 #
 #   Required keys:
 #       path                Location to store the database (all types)
 #
 #   Optional keys:
-#       compression         0 for none, 1 for Snappy compression
+#
+#       These keys may be configured for any type of backend:
```
Comment (rec) [inline doc/rippled-example.cfg]: This puzzled me for a bit.  But the keys themselves aren't "configured".  What you mean is that the following keys are [valid, legal, acceptable] for any backend, yes?

File: doc/rippled-example.cfg [hunk 3]
```diff
@@ -641,15 +660,12 @@
...
-#       The 'temp_db' configures a look-aside cache for high volume storage
-#           which doesn't necessarily persist between server launches. This
-#           is an optional configuration parameter. If it is left out then
-#           no look-aside database is created or used.
-#
 #       The 'import_db' is used with the '--import' command line option to
 #           migrate the specified database into the current database given
 #           in the [node_db] section.
 #
+#   [temp_db]       Settings for the look-aside temporary db (optional)
```
Comment (rec) [inline doc/rippled-example.cfg]: Is this "look-aside" concept defined already in this file?

File: doc/rippled-example.cfg [hunk 4]
```diff
@@ -606,30 +606,49 @@
...
-#       path=db/rocksdb
+#   Example:
+#       type=nudb
+#       path=db/nudb
+#
+#   The "type" field must be present and controls the choice of backend:
+#
+#   type = NuDB
+#
+#       NuDB is high performance, but only works with Solid State (SSD)
```
Comment (rec) [inline doc/rippled-example.cfg]: I'd say a little here about what NuDB is.  "NuDB is a high-performance database written by Ripple Labs and optimized for rippled's use cases." or something like that.

File: doc/rippled-example.cfg [hunk 5]
```diff
@@ -606,30 +606,49 @@
...
+#   type = RocksDB
+#
+#       This can work with spinning disks or installations that have
+#       lower IOPS (read/write throughput). It is not suitable for
+#       storing full history because it slows down as the data set
+#       increases. If RocksDB is chosen as the backend, we suggest
+#       also setting up online delete. RocksDB is not available
+#       on Windows environments.
+#
+#       The RocksDB backend also provides these optional parameters:
```
Comment (rec) [inline doc/rippled-example.cfg]: Similarly, I'd say a tiny bit about what RocksDB is, though they can always look that up.

File: doc/rippled-example.cfg (hunk 2 above)
Comment (vinniefalco) [inline doc/rippled-example.cfg]: key/value pairs, see line 66:
https://github.com/vinniefalco/rippled/blob/develop/doc/rippled-example.cfg#L66

File: doc/rippled-example.cfg (hunk 1 above)
Comment (vinniefalco) [inline doc/rippled-example.cfg]: Oops...I later reversed the order of backend listings.

**Pull request closed without merging**